doc/release-notes/0.9/0.9.md

```diff
@@ -0,0 +1,184 @@
+<td valign="top">GPU acceleration with `java.util.Array.sort()` is not supported</td>
+<td valign="top">Linux and Windows platforms</td>
+<td valign="top">GPU acceleration on `sort()` operations is not available.</td>
+<td valign="top">None, although `com.ibm.gpu.Maths` can be used directly.</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/897">#897</a></td>
+<td valign="top">Javadoc not building some OpenJ9 extensions.</td>
+<td valign="top">All platforms</td>
+<td valign="top">Some API documentation is missing in the <code>build/&lt;platform&gt;/docs</code> directory.</td>
+<td valign="top">None</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/898">#898</a></td>
+<td valign="top">Javadoc building some unsupported OpenJ9 extensions.</td>
+<td valign="top">All platforms</td>
+<td valign="top">Some API documentation in <code>build/&lt;platform&gt;/docs</code> is not supported with OpenJ9.</td>
+<td valign="top">None</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/ibmruntimes/openj9-openjdk-jdk8/issues/17">#17</a></td>
+<td valign="top">Builds contain extra shared libraries.</td>
+<td valign="top">All platforms</td>
+<td valign="top">Shared libraries must not be used in production.</td>
+<td valign="top">None</td>
+</tr>
+</tbody>
+</table>
+
+
+## Functional verification tests
+The following issues relate to test failures:
+
+- [#238](https://github.com/eclipse/openj9/issues/238), [#244](https://github.com/eclipse/openj9/issues/244), [#362](https://github.com/eclipse/openj9/issues/362), [#366](https://github.com/eclipse/openj9/issues/366), [#480](https://github.com/eclipse/openj9/issues/480), [#858](https://github.com/eclipse/openj9/issues/858), [#968](https://github.com/eclipse/openj9/issues/968), [#1127](https://github.com/eclipse/openj9/issues/1127), [#1128](https://github.com/eclipse/openj9/issues/1128), [#1129](https://github.com/eclipse/openj9/issues/1129), [#1130](https://github.com/eclipse/openj9/issues/1130), [#1131](https://github.com/eclipse/openj9/issues/1131), [#1144](https://github.com/eclipse/openj9/issues/1144), [#1151](https://github.com/eclipse/openj9/issues/1151), [#1203](https://github.com/eclipse/openj9/issues/1203), [#1204](https://github.com/eclipse/openj9/issues/1204), [#1205](https://github.com/eclipse/openj9/issues/1205), [#1207](https://github.com/eclipse/openj9/issues/1207), [#1209](https://github.com/eclipse/openj9/issues/1209), [#1212](https://github.com/eclipse/openj9/issues/1212), and [#1244](https://github.com/eclipse/openj9/issues/1244).
+
+Because some tests are not yet enabled, functional problems might not be exposed. In addition, some IBM authored functional verification testing is not yet contributed to Eclipse, such as issue  [#724](https://github.com/eclipse/openj9/issues/724).
+
+## Other changes
+
+The following issues are addressed in this release:
+
+- add all other # numbers that closed between 0.8.0 and 0.9.0?
```

Comment:
This looks like a TODO that still needs to be handled.  Sue had mentioned a changes list based using Node's release tool....

Reply:
Yep - I raised https://github.com/eclipse/openj9/issues/1979. No time to try and code this myself and it's not moved on. 

Suggest we update the "other changes" section to tell people to run 

`git log openj9-0.8.0..openj9-0.9.0`

.. until we have a tool that can generate a nice concise list.